### PR TITLE
Add ignore: ["first-nested"] to declaration-empty-line-before

### LINKED ISSUE #3101: Add ignore: ["first-nested"] to declaration-empty-line-before
This option would be useful for when using prettier and stylelint side-by-side (see: https://github.com/prettier/prettier/issues/2621)

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Closes #3101 

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

**Comment by ntwb** [comment]
One thought, is it worth adding a note in the readme for this and #3104 that these are to help facilitate the use of Prettier?

**Comment by hudochenkov** [comment]
> One thought, is it worth adding a note in the readme for this and #3104 that these are to help facilitate the use of Prettier?

I don't think it's a good idea. There is no reason promote tools in rules description.

File: lib/rules/declaration-empty-line-before/README.md [hunk 1]
```diff
@@ -268,6 +268,21 @@ a {
...
+Ignore declarations that are nested and the first child of their parent node.
+
+For example, with `"always"`:
+
+The following patterns are *not* considered violations:
+
+```css
+a {
+  bottom: 15px;
+  top: 5px;
```
Comment (hudochenkov) [inline lib/rules/declaration-empty-line-before/README.md]: There should be an empty line before.

Comment (jeddy3) [reply lib/rules/declaration-empty-line-before/README.md]: Whoops.

**Comment by jeddy3** [comment]
Fixed

**Comment by jeddy3** [comment]
> -   Added: `ignore: ["first-nested"]` to `declaration-empty-line-before` ([#3103](https://github.com/stylelint/stylelint/pull/3103)).

**Pull request merged**